DP03_05_Using_User-Supplied_Catalogs.ipynb

```diff
@@ -0,0 +1,683 @@
+{
```

Comment:
<p>flake8 complained about the following:</p>

<pre class="ql-syntax" spellcheck="false">pycodestyle INFO: 2:47: W291 trailing whitespace
pycodestyle INFO: 47: E203 whitespace before ':'
pycodestyle INFO: 13:80: E501 line too long (81 &gt; 79 characters)
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rubin-dp0/tutorial-notebooks/pull/187/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='45'/>